File: epi_judge_python/collatz_checker.py
from test_framework import generic_test


def collatz_conjecture(n: int) -> bool:
    # TODO - you fill in here.
    return False


if __name__ == '__main__':
    exit(
        generic_test.generic_test_main('collatz_checker.py',
                                       'collatz_checker.tsv',
                                       collatz_conjecture))

File: epi_judge_java/epi/CollatzChecker.java
package epi;
import epi.test_framework.EpiTest;
import epi.test_framework.GenericTest;
public class CollatzChecker {
  @EpiTest(testDataFile = "collatz_checker.tsv")

  public static boolean testCollatzConjecture(int n) {
    // TODO - you fill in here.
    return false;
  }

  public static void main(String[] args) {
    System.exit(
        GenericTest
            .runFromAnnotations(args, "CollatzChecker.java",
                                new Object() {}.getClass().getEnclosingClass())
            .ordinal());
  }
}
